tests/unit_tests/test_game_rating.py

```diff
@@ -277,17 +314,22 @@ async def test_on_game_end_ladder_ratings_published(ladder_game, players):
     assert players.hosting.id in results
     assert players.joining.id in results
 
-    host_result = results[players.hosting.id]
-    assert host_result["rating_type"] == RatingType.LADDER_1V1
-    assert "new_rating_mean" in host_result
-    assert "new_rating_deviation" in host_result
-    assert host_result["outcome"] == GameOutcome.VICTORY.value
+    host_ladder_result, host_global_result = results[players.hosting.id]
+    assert host_ladder_result["rating_type"] == RatingType.LADDER_1V1
+    assert "new_rating_mean" in host_ladder_result
+    assert "new_rating_deviation" in host_ladder_result
+    assert host_ladder_result["outcome"] == GameOutcome.VICTORY.value
 
-    join_result = results[players.joining.id]
-    assert join_result["rating_type"] == RatingType.LADDER_1V1
-    assert "new_rating_mean" in join_result
-    assert "new_rating_deviation" in join_result
-    assert join_result["outcome"] == GameOutcome.DEFEAT.value
+    assert host_global_result["rating_type"] == RatingType.GLOBAL
+    assert "new_rating_mean" in host_global_result
+    assert "new_rating_deviation" in host_global_result
+    assert host_global_result["outcome"] == GameOutcome.VICTORY.value
+
+    join_ladder_result, = results[players.joining.id]
+    assert join_ladder_result["rating_type"] == RatingType.LADDER_1V1
+    assert "new_rating_mean" in join_ladder_result
+    assert "new_rating_deviation" in join_ladder_result
+    assert join_ladder_result["outcome"] == GameOutcome.DEFEAT.value
```

Comment:
It's also left out here and on following tests. I feel like I am missing something...
The adjustment should happen for both players, shouldn't it?

Reply:
Only if the displayed rating increases.